packages/flutter_tools/test/commands.shard/hermetic/downgrade_test.dart

```diff
@@ -0,0 +1,169 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Can you add a test of downgrading to a hotfix (branched) version, then upgrading again?

Reply:
How about:

1. Start at specified hotfix version on stable.
2. Upgrade to newest stable.
3. Downgrade back to original hotfix?